### ISSUE TITLE
Monkey-patch for Winston3 publishes too much

### ISSUE THREAD

**Issue by EJBroeders** [microsoft/node-diagnostic-channel#115]
The Winston patch publishes on the diagnostic channel all logs that Winston collects. I would expect as an end-user that only messages above the configured Winston log level get published.

Current situation:
```
import * as appInsights from 'applicationinsights';
import * as winston from 'winston';

const logger = winston.createLogger({
  level: 'warn',
});

appInsights.setup();
appInsights.start();

logger.debug('This should not trigger a publish on the diagnostic-channel - but it does');
```

Links:
- File which should be patched: https://github.com/microsoft/node-diagnostic-channel/blob/master/src/diagnostic-channel-publishers/src/winston.pub.ts
- README to Winston on log levels: https://github.com/winstonjs/winston#logging-levels
